tests/compile-test.rs

```diff
@@ -162,8 +163,8 @@ fn base_config(test_dir: &str) -> compiletest::Config {
 }
 
 fn run_ui() {
-    let config = base_config("ui");
-    // use tests/clippy.toml
```

Comment:
I'd like to keep this comment since it explains what the `CARGO_MANIFEST_DIR` `VarGuard` is for

Reply:
Apologies. I thought I introduced that comment by mistake.